Sources/KituraNet/FastCGI/FastCGIServerRequest.swift

```diff
@@ -55,7 +55,7 @@ public class FastCGIServerRequest: ServerRequest {
     /// The URL from the request in UTF-8 form
     /// This contains just the path and query parameters starting with '/'
     /// Use 'urlURL' for the full URL
-    public var url: Data { return requestUri?.data(using: .utf8) ?? Data() }
+    public var url: Data { return Data(urlURL.absoluteString.utf8) }
```

Comment:
Why was this changed to use `urlURL` rather than `requestUri`? The comment above says they are different.

Reply:
@ianpartridge Thanks for finding this. I've fixed it.